From a77e89e36d254dce2086fe3c70cbdb0da0298e12 Mon Sep 17 00:00:00 2001
From: Pauli Nieminen <pauli.nieminen@linux.intel.com>
Date: Tue, 15 Nov 2011 20:33:05 +0200
Subject: Fix build problem when including hal and egl to same c file

If including hal and egl headers to same c file compilation files
because ANativeWindow type is defined in two headers. In HC egl didn't
include system/core/include/system/window.h which hid the type conflict.

Change-Id: Icaf7b29f913f3d7fa8f6d70bb196585d22e7d05c
Original-Change-Id: Icd37329d80fb4e8b9baf9015c9b00e43ebc4f76b
Signed-off-by: Pauli Nieminen <pauli.nieminen@linux.intel.com>
---
 include/android/native_window.h |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/android/native_window.h b/include/android/native_window.h
index 2f4f2d3..43fe78b 100644
--- a/include/android/native_window.h
+++ b/include/android/native_window.h
@@ -19,6 +19,8 @@
 
 #include <android/rect.h>
 
+#include <system/window.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
@@ -32,9 +34,6 @@ enum {
     WINDOW_FORMAT_RGB_565            = 4,
 };
 
-struct ANativeWindow;
-typedef struct ANativeWindow ANativeWindow;
-
 typedef struct ANativeWindow_Buffer {
     // The number of pixels that are show horizontally.
     int32_t width;
-- 
1.7.4.1

